api/client/samples/analogous_years/lib/final_ranks_computation.py

```diff
@@ -51,9 +50,10 @@ def enso_data(start_date):
     return enso_data_series
 
 
-def get_file_name(api_token, data_series_list, initial_date, final_date):
+def get_file_name(api_token, data_series_list, initial_date, final_date,
+                  api_host='api.gro-intelligence.com'):
```

Comment:
to accommodate the scenario when the default host has to change again, you can still have a global var, sth like `DEFAULT_API_HOST = 'api.gro-intelligence.com'`, and default `api_host=DEFAULT_API_HOST`, to avoid changing the host address in every single call.

Reply:
Done